java/vector/src/main/java/org/apache/arrow/vector/table/Row.java

```diff
@@ -98,6 +102,8 @@
 import org.apache.arrow.vector.holders.NullableUInt8Holder;
 
 /**
+ * TODO: Modify the getters for Duration and others so that they return something better than ArrowBuf when possible
```

Comment:
Did you mean to leave this TODO in here? (It should perhaps be linked to an issue?)

Reply:
removed obsolete TODOs